syntheseus/search/node_evaluation/base.py

```diff
@@ -93,10 +93,25 @@ def __init__(
         self,
         return_log: bool,
         return_negated: bool,
+        normalize: bool = False,
         temperature: float = 1.0,
         clip_probability_min: float = 1e-10,
         clip_probability_max: float = 0.999,
     ) -> None:
+        """Initialized the evaluator.
+
+        Args:
+            return_log: Whether to return the logarithm of the probability instead of the
+                probability itself.
+            return_negated: Whether to multiply the result by `-1`.
```

Comment:
Taking a second look at this: is there a reason we have a `return_negated` option instead of downstream classes just doing `-np.asarray(outputs).tolist()`?

Reply:
By "downstream classes", do you mean child classes of this, i.e. `ReactionModelLogProbCost`?

Reply:
Yes, exactly.

Reply:
Then there's no strong reason, negation could also happen in the child class. The only argument for having it in the base class would be if there were many classes applying this negation; then this would reduce some duplication as otherwise each of these subclasses would have to override `_evaluate_nodes` in the same way. But currently there is only one such class and maybe there won't be any others, so I can move the negation there.